src/StyleTransfer/index.js

```diff
@@ -160,6 +209,18 @@ class StyleTransfer extends Video {
     return result;
   }
 
+  /**
+   * Dispose of all Tensors in instance properties.
+   * @public
+   * @return {void}
+   */
+  dispose() {
+    Object.values(this.variables).forEach(variable => variable.dispose());
+    this.timesScalar.dispose();
+    this.plusScalar.dispose();
+    this.epsilonScalar.dispose();
+  }
+
```

Comment:
Positive Affirmation:
* I totes dig this. Feels also like a nice api convention that helps us to keep an eye on the tensors that might otherwise get lost. 
* In other places we tried this "dispose tensors" flag that was set to true by default, but people could disable that if they wanted to handle this on their own but we can totally revisit this too https://github.com/ml5js/ml5-library/blob/main/src/BodyPix/index.js#L52

Reply:
Yeah I've seen that `returnTensors` option in a bunch of the models which generate an image.  We use that to either keep or dispose of the tensor for the generated image.

The issue here is the tensors which are used in the model itself.  The equivalent for BodyPix is calling `this.model.dispose()` to dispose of the underlying bodypix model.  We don't do that anywhere for most of our models.  The exceptions are [Neural Network](https://github.com/ml5js/ml5-library/blob/6c3d0e8ea43f48e73bd0b50497cb3e171e966089/src/NeuralNetwork/NeuralNetwork.js#L281) and [KNNClassifier](https://github.com/ml5js/ml5-library/blob/6c3d0e8ea43f48e73bd0b50497cb3e171e966089/src/KNNClassifier/index.js#L169) where we do have a `dispose()` method already. It has to be user-initiated so that we don't kill the model before they are done with it.